code/game/gamemodes/changeling/traitor_chan.dm

```diff
@@ -54,10 +54,11 @@
 
 /datum/game_mode/traitor/changeling/post_setup()
 	for(var/datum/mind/changeling in changelings)
-		changeling.current.make_changeling(changeling.current)
+		changeling.current.make_changeling()

```

Comment:
changeling.current.make_changeling(changeling.current) isn't needed, in fact, that was the only place in the source that make_changeling() was used in that way.  So its just a consistency update.